<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationWindow</class>
 <widget class="QMainWindow" name="ApplicationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ApplicationWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="LaserGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>651</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>20</width>
      <height>651</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>821</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>821</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="AzimuthLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Azimuthal Freq :</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalPointsLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Points : </string>
    </property>
   </widget>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RE05 Lidar Sensor Simulation 
 Control Panel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scanLineLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ScanLines : </string>
    </property>
   </widget>
   <widget class="QLabel" name="maxRange">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Range : </string>
    </property>
   </widget>
   <widget class="QLabel" name="camYLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Y : </string>
    </property>
   </widget>
   <widget class="QLabel" name="ModeLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Mode :</string>
    </property>
   </widget>
   <widget class="QLabel" name="samplingFreqLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling Freq : </string>
    </property>
   </widget>
   <widget class="QLabel" name="camXLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Status : </string>
    </property>
   </widget>
   <widget class="QLabel" name="camZLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Laser Z : </string>
    </property>
   </widget>
   <widget class="QTabWidget" name="LaserWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>230</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="startstopTab">
     <attribute name="title">
      <string>Start/Stop Laser</string>
     </attribute>
     <widget class="QPushButton" name="startLaserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Laser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopLaserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Laser</string>
      </property>
     </widget>
     <widget class="QLabel" name="AzimuthalLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Azimuthal : </string>
      </property>
     </widget>
     <widget class="QLabel" name="scanLinesLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan Lines : </string>
      </property>
     </widget>
     <widget class="QLabel" name="AzimuthalFullScanLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>46</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="ScanLineFullScanLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>46</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>64</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="laserPositionTab">
     <attribute name="title">
      <string>Laser Position Parameters</string>
     </attribute>
     <widget class="QPushButton" name="LaserPositionButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QLabel" name="laserXLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser X :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LaserXvLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser Xv :</string>
      </property>
     </widget>
     <widget class="QLabel" name="laserYLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser Y :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LaserYvLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser Yv :</string>
      </property>
     </widget>
     <widget class="QLabel" name="laserZLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser Z :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LaserZvLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Laser Zv :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserXSpinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserYSpinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserZSpinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserXvSpinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserYvSpinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LaserZvSpinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="ScanWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="fullScanTab">
     <attribute name="title">
      <string>Full Scan Mode</string>
     </attribute>
     <widget class="QPushButton" name="fullScanModeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="scanLinesLabel_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan Lines : </string>
      </property>
     </widget>
     <widget class="QLabel" name="AzimuthalLabel_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Azimuthal : </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="scanLineSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="AzimuthalspinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="boundedElevationTab">
     <attribute name="title">
      <string>Bounded Elevation</string>
     </attribute>
     <widget class="QPushButton" name="boundedElevationModeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="UpperBound">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper Bound :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LowerBound">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower Bound :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="UpperBoundSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LowerBoundSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="regionScanTab">
     <attribute name="title">
      <string>Region Scan</string>
     </attribute>
     <widget class="QPushButton" name="regionScanModeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QLabel" name="UpperBound_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper 
Bound :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LowerBound_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower 
Bound :</string>
      </property>
     </widget>
     <widget class="QLabel" name="UpperBound_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Angular 
Left :</string>
      </property>
     </widget>
     <widget class="QLabel" name="LowerBound_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Angular 
Right :</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="UpperBoundRegionSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="LowerBoundRegionSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="AngularLeftSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>45.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="AngularRightSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="samplingFreqValueLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>70</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalPointsValueLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>100</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxRangeValueLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>130</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="AzimuthalFreqLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="scanLinesValueLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>190</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LaserXValueLabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>70</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LaserYValueLabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>100</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LaserZValueLabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>130</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="scanModeValueLabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>160</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="LaserStatusLabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>190</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LaserGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">laserglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
